doc/website/getting-started/overview.md

```diff
@@ -0,0 +1,540 @@
+need a longer lifetime, we have to copy or move the data from the ``sample``.
+
+This only allows us to get one sample at a time. Should we want to get all currently available samples we can do so
+by using an additional loop.
+
+```
+while(keepRunning) {
+    while(subscriber.hasNewSamples()) {
+        subscriber->take()
+            .and_then([](iox::popo::Sample<const CounterTopic>& sample) {
+                CounterTopic* ptr = sample.get();
+                /* process the received data using the ptr */
+
+                /* alternatively use operator-> */
+                uint32_t counter = sample->counter;           
+            })
+            .or_else([](iox::popo::ChunkReceiveError) { /* handle the error */ });
+    }
+    // wait for new samples (either sleep or use the waitset)
+}
+```
+Here we do not check whether we actually have data since we already know there is data available by calling
+``hasNewSamples``.
+
+### Untyped API
+
+The untyped API offeres similar capabilities and is hence usable in a similar way. The major difference is that
+neither publisher nor subscriber have any knowledge about the underlying type they send or receive. This means that
+the user is responsible to ensure the data is read correctly, i.e. there is no type safety guaranteed by the API
+itself.
+
+#### Creating a publisher
+
+When creating an untyped publisher we do not need to specify a data type as template paraemter.
+
+```
+iox::popo::UntypedPublisher publisher({"Group", "Instance", "CounterTopic"});
+publisher.offer();
+```
+#### Sending data
+
+1. Loan and emplace
+
+    Before sending, we have to loan a chunk of memory to emplace our data.
+    ```
+    auto result = publisher.loan(sizeof(CounterTopic));
+    ```
+    Since the data type is not known to the publisher, we have to provide the size in bytes of the payload data we
+    intend to send.
+    
+    If we successfully acquired a chunk, we can construct the data to be send using emplacement new and publish it.
+
+    ```
+    if (!result.has_error())
+    {
+        auto& sample = result.value();
+        new (sample.get()) CounterTopic(73);
+        sample.publish();
+    } else 
+    {
+        //could not acquire chunk, handle the error
+    }
+    ```
+
+    Here emplacement new is required, there is no preconstructed object at ``sample.get()``.
+
+2. Functional approach with loaning
+
+    ```
+    publisher.loan(sizeof(CounterTopic))
+            .and_then([&](iox::popo::Sample<void>& sample) { 
+                new (sample.get()) CounterTopic(73);
+                sample.publish();
+            })
+            .or_else([](iox::popo::AllocationError) {
+                /* handle the error */
+            });
+    ```
+    Notice that we get an untyped sample, ``iox::popo::Sample<void>`` (we could also use ``auto& sample`` in the
+    lambda arguments to shorten it). Again we access the pointer to the underlying raw memory of the sample and
```

Comment:
You might know my opinion on this already :-P Please either remove the braces or remove the sentence

Reply:
Here I would find them justified (optional information but useful when wokring with the API), but I do not really care. I create a new sentence.